bpmn/multi-instance-process.bpmn

```diff
@@ -0,0 +1,209 @@
+  <bpmn:process id="multi-instance-process" isExecutable="true">
+    <bpmn:extensionElements />
+    <bpmn:startEvent id="order-placed" name="Order placed">
+      <bpmn:outgoing>Flow_02fbvun</bpmn:outgoing>
+    </bpmn:startEvent>
+    <bpmn:intermediateCatchEvent id="payment-received" name="Payment received">
+      <bpmn:incoming>Flow_02m51mh</bpmn:incoming>
+      <bpmn:outgoing>Flow_05c4box</bpmn:outgoing>
+      <bpmn:messageEventDefinition id="MessageEventDefinition_0bjp0jk" messageRef="Message_3mcrbl7" />
+    </bpmn:intermediateCatchEvent>
+    <bpmn:exclusiveGateway id="check-value" name="Order value?" default="Flow_1bfyuha">
+      <bpmn:incoming>Flow_05c4box</bpmn:incoming>
+      <bpmn:outgoing>Flow_0c047h2</bpmn:outgoing>
+      <bpmn:outgoing>Flow_1bfyuha</bpmn:outgoing>
+    </bpmn:exclusiveGateway>
+    <bpmn:sequenceFlow id="Flow_05c4box" sourceRef="payment-received" targetRef="check-value" />
+    <bpmn:serviceTask id="ship-with-insurance" name="Ship with insurance">
+      <bpmn:extensionElements>
+        <zeebe:taskDefinition type="ship-with-insurance" />
+      </bpmn:extensionElements>
+      <bpmn:incoming>Flow_0c047h2</bpmn:incoming>
+      <bpmn:outgoing>Flow_1ua42cy</bpmn:outgoing>
+    </bpmn:serviceTask>
+    <bpmn:sequenceFlow id="Flow_0c047h2" sourceRef="check-value" targetRef="ship-with-insurance">
+      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">=orderValue&gt;=100</bpmn:conditionExpression>
+    </bpmn:sequenceFlow>
+    <bpmn:serviceTask id="ship-without-insurance" name="Ship without insurance">
+      <bpmn:extensionElements>
+        <zeebe:taskDefinition type="ship-without-insurance" />
+      </bpmn:extensionElements>
+      <bpmn:incoming>Flow_1bfyuha</bpmn:incoming>
+      <bpmn:outgoing>Flow_0o34bea</bpmn:outgoing>
+    </bpmn:serviceTask>
+    <bpmn:sequenceFlow id="Flow_1bfyuha" name="&#60;100" sourceRef="check-value" targetRef="ship-without-insurance" />
+    <bpmn:exclusiveGateway id="Gateway_09paus8">
+      <bpmn:incoming>Flow_1ua42cy</bpmn:incoming>
+      <bpmn:incoming>Flow_0o34bea</bpmn:incoming>
+      <bpmn:outgoing>Flow_0kjm2o6</bpmn:outgoing>
+    </bpmn:exclusiveGateway>
+    <bpmn:sequenceFlow id="Flow_1ua42cy" sourceRef="ship-with-insurance" targetRef="Gateway_09paus8" />
+    <bpmn:sequenceFlow id="Flow_0o34bea" sourceRef="ship-without-insurance" targetRef="Gateway_09paus8" />
+    <bpmn:endEvent id="Event_01079cq" name="Order fulfilled">
+      <bpmn:incoming>Flow_0kjm2o6</bpmn:incoming>
+    </bpmn:endEvent>
+    <bpmn:sequenceFlow id="Flow_0kjm2o6" sourceRef="Gateway_09paus8" targetRef="Event_01079cq" />
+    <bpmn:task id="parse-order-items" name="Parse order items">
+      <bpmn:incoming>Flow_02fbvun</bpmn:incoming>
+      <bpmn:outgoing>Flow_0kk1pnx</bpmn:outgoing>
+      <bpmn:multiInstanceLoopCharacteristics>
```

Comment:
As mentioned in acceptance criteria, it would be good to have a multi-instance process example here as well. Multi-instance is an activity which is executed multiple times and service tasks, receive tasks, embedded subprocesses and call activities can be marked as a multi-instance task (source https://docs.camunda.io/docs/components/modeler/bpmn/multi-instance/)
I don't see here any activity that would be executed multiple times, although this task is marked as a multi-instance task.